### ISSUE TITLE
bag coefficient documentation

### ISSUE THREAD

**Issue by bobwmcgrath** [mit-drl/e-vent#65]
If I understand correctly, there are 3 coefficients that determine the particular profile for different ambu bags but I could use a better explanations for each number so I can make things work with my ambu bag.  These coefficients are in constants.h

// Bag Calibration for AMBU Adult bag
const struct {float a, b, c;} COEFFS{1.29083271e-03, 4.72985182e-01, -7.35403067e+01};

**Comment by amadoantonini** [comment]
Yes, we should document this better. They are simply the polynomial coefficients of a second degree fit to volume-vs-ticks data for that particular bag. See [here](https://github.com/mit-drl/e-vent/blob/e5286cf4501ea58fcaeb0cd2051117449c81f9a4/Utilities.cpp#L53).

**Comment by bobwmcgrath** [comment]
I had seen where they were being used, but I'm not sure how to change those numbers to fit my bag.  Were the original numbers determined by measuring the volume change as the machine squeezed the bag?

**Comment by amadoantonini** [comment]
That's right! We'll add more documentation on that when we get a chance.

**Comment by RubenPV591** [comment]
Hi, I've been working in a mathematical model for the Bag valve mask volume versus claw movement, I didn't knew where to share it in the group, so I'm publishing it here in case could be useful.  If somebody knows if I should share this in another site just tell me please.

Greetings.

**Comment by RubenPV591** [comment]
Hi, I've been working in a mathematical model for the Bag valve mask volume versus claw movement, I didn't knew where to share it in the group, so I'm publishing it here in case could be useful. If somebody knows if I should share this in another site just tell me please.

Greetings.

[volumenAmbu02eng.pdf](https://github.com/mit-drl/e-vent/files/4920915/volumenAmbu02eng.pdf)
